docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_updateManagementCenter/_index.md

```diff
@@ -0,0 +1,180 @@
+
+- Go to your **resource group** and click on **Policies**:
+
+    ![Resource Group - Policies](./05.png)
+
+- You will see the *(jumpstart) Configure periodic checking for missing system updates on azure Arc-enabled servers* policy assigned. Then, click on **Remediation**:
+
+    ![Policy for checking missing updates](./06.png)
+
+- Right click on the policy *(jumpstart) Configure periodic checking for missing system updates on azure Arc-enabled servers* policy assigned. Then, click on **Remediate**:
+
+    ![Policy for checking missing updates - Remediation](./07.png)
+
+- You will see your Azure Arc-enabled server as a resource to be remediated. Click on the **Remediate** button:
+
+    ![Policy for checking missing updates - trigger remediation](./08.png)
+
+- You can see the status of the **Remediation task** and the number of resources remediated:
+
+    ![Policy for checking missing updates - remediation completed](./09.png)
+
+  > **NOTE: You will need to wait some time until the first update assessment is run.**
+
+- On the update assessment task is run, go back to your **Azure Arc-enabled server** and click on **Extensions**. You will see that a new extension has been installed. Update management center relies on this Azure extension designed to provide all the functionality required to interact with the operating system to manage the assessment and application of updates.
+
+    ![Update Management Center extension](./10.png)
+
+- Click on **Updates**. Under **Recommended updates**, you will see which are the missing updates on this machine:
+
+    ![Missing updates](./11.png)
+
+- If you click on the **History** tab, you will see the details related to the updates assessment and operations:
+
+    ![Update assessment history](./12.png)
+
+## Update Deployment
+
+On the other hand, as part of this scenario, a new **Maintenance Configuration** resource is created. It allows you to schedule recurring updates deployments on your Azure Arc-enabled servers.
+
+- Go to your **resource group** and click on the new **Maintenance Configuration** resource:
+
+    ![New maintenance configuration resource](./13.png)
+
+- Review the following **Maintenance Configuration** settings: **Machines**, which includes the Azure Arc-enabled server; **Schedule** of the maintenance configuration resource (it will start one day after you deploy this scenario); and **Updates** settings:
```

Comment:
```suggestion
- Review the following **Maintenance Configuration** settings: **Machines**, which includes the Azure Arc-enabled server; **Schedule** of the maintenance configuration resource (it will start one day after you deploy this scenario), and **Updates** settings:
```